### PR TITLE
ImageBigData: distinguish between no-such-image and no-such-item

### PR DESCRIPTION
**Pull request by nalind**
When reporting an error from store.ImageBigData(), distinguish between cases where we can't find the specified image, and where we found the image, but it didn't have a matching requested item.

### CODE REVIEW

File: store.go [hunk 1]
```diff
@@ -1515,6 +1516,12 @@ func (s *store) ImageBigData(id, key string) ([]byte, error) {
 		if err == nil {
 			return data, nil
 		}
+		if store.Exists(id) {
+			foundImage = true
+		}
+	}
+	if foundImage {
```
Comment (TomSweeneyRedHat) [inline store.go]: should this be `if !foundImage {`  ?

Comment (nalind) [reply store.go]: That wouldn't match the error that we return in the block.

Comment (TomSweeneyRedHat) [reply store.go]: ugh, coulda sworn that said 'error locating image...'  Darn dyslexia!  Thx, all good.

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**